src/types.h

```diff
@@ -314,7 +314,11 @@ typedef struct
             char value[VALUE_SIZE];
             char truncated;
         } v;
-        u64 retcode;
+        struct
```

Comment:
Why put it in a struct instead of just adding another field?

Reply:
I'm not crazy about this, but because this change 'replaces' the retcodes for the `clone`/`fork` family calls, I wanted to keep them together somehow. As in, the retcode was _some value_ and but the true `pid` is _some other value_. We probably want a way to grab both the true PID and the namespaced PID.

Reply:
That's fair. If they are mutually exclusive though (we always use one or the other) wouldn't a union make sense? Unless there is a case where we would want both. Either way I don't think it is a huge deal. Just seemed a bit odd at first glance.

Reply:
Even though this should be safe, unions are _discouraged_ by Rust. The use case for a struct would be returning the `fork` retcode (child PID in namespace) alongside the true PID. Although that would require two different probes, with two different messages, so a union is probably more appropriate. 

I think I just talked myself into a union. 567dc34aff74f0aad82406df6b0a206cca61933c